src/Generator.php

```diff
@@ -119,7 +119,7 @@ public function __construct(array $settings = null)
 	 */
 	public function setSecure($secure = true)
 	{
-		$this->secure = (bool) $secure;
+		$this->secure = (bool)$secure;
```

Comment:
I would agree. It looks neater with a space too, so I will revert it.

The other issue is with the alignment of PHPDoc comments and variables. Personally I think aligning them is a waste of space and doesn't help much with readability. @JN-Jones likes them though :stuck_out_tongue: